app/javascript/pages/admin/Admin/index.js

```diff
@@ -87,24 +87,15 @@ const OfficeFilter = ({ adminOfficeFilter, changeAdminOfficeFilter, offices }) =
 
 class Admin extends Component {
   componentDidUpdate() {
-    const {
-      history,
-      data: { networkStatus, currentUser },
-    } = this.props
+    const { history, data: { networkStatus, currentUser } } = this.props
 
     if (networkStatus !== NetworkStatus.loading && !currentUser.isAdmin) {
       history.push('/portal')
     }
   }
 
   render() {
-    const {
-      routing,
-      children,
-      adminOfficeFilter,
-      changeAdminOfficeFilter,
-      data: { currentUser, offices },
-    } = this.props
+    const { routing, children, adminOfficeFilter, changeAdminOfficeFilter, data: { currentUser, offices } } = this.props
```

Comment:
did prettier single line this? The line seems pretty long.

Reply:
Yes, prettier did this. I could commit with `--no-verify` if this line is annoying and/or put up a separate PR to decrease the max line length setting of the linter.

Reply:
nah, I trust our prettier config. It was just surprised.